docker-app/qfieldcloud/urls.py

```diff
@@ -29,6 +29,7 @@
 )
 from qfieldcloud.authentication import views as auth_views
 from qfieldcloud.core.views import files_views
+
```

Comment:
Please revert this change, it brings only noise without additional value.

Reply:
Done

Reply:
It is still there.

Reply:
Done!